internal/pkg/template/templates/environment/partials/s3-elb-access-logs.yml

```diff
@@ -0,0 +1,39 @@
+ELBAccessLogsBucketPolicy:
+  Type: AWS::S3::BucketPolicy
+  Condition: CreateALB
+  DependsOn: ELBAccessLogsBucket
+  Properties:
+    Bucket: !Ref ELBAccessLogsBucket
+    PolicyDocument:
+      Version: '2012-10-17'
+      Statement:
+        - Action:
+            - s3:PutObject
+          Effect: Allow
+          Resource:
+            - !Join
+              - ''
+              - - 'arn:'
+                - !Ref AWS::Partition
+                - ':s3:::'
+                - !Ref ELBAccessLogsBucket
+                - '/'
+                - {{ .BucketPrefix }}
+                - '/AWSLogs/'
+                - !Ref AWS::AccountId
+                - '/*'
+          Principal:
+            AWS: !Join [ "", [  "arn:aws:iam::", !FindInMap [ RegionalConfigs, !Ref 'AWS::Region', ELBAccountId ], ":root" ] ]
+ELBAccessLogsBucket:
+  Type: AWS::S3::Bucket
+  Condition: CreateALB
+  Properties:
+    {{- if .BucketName}}
+    BucketName: {{ .BucketName }}
+    {{- end }}
```

Comment:
We can remove this as we should only create the bucket if the user doesn't use the `bucket_name` field

Reply:
Yeah right make sense, I had this block before, when we were still discussing if we want to provide an option to customers to create a bucket or not?